Sources/TSCUtility/PkgConfig.swift

```diff
@@ -163,13 +174,21 @@ public struct PkgConfig {
             var libs = [String]()
             
             for dep in dependencies {
+                let firstOccurrence = loadingContext.pkgConfigStack.firstIndex(of: dep)
+
+                if let index = firstOccurrence {
```

Comment:
If `firstOccurrence` isn't needed elsewhere, is it possible to simplify this to just `if loadingContext.pkgConfigStack.contains(dep)`?